service/controller/clusterapi/v27/key/key.go

```diff
@@ -122,6 +122,10 @@ func ELBNameIngress(cluster v1alpha1.Cluster) string {
 	return fmt.Sprintf("%s-ingress", ClusterID(cluster))
 }
 
+func ImageID(cluster v1alpha1.Cluster) string {
+	return imageIDs()[Region(cluster)]
+}
```

Comment:
I removed the error from the interface. When we should ever have a problem with regions being misconfigured, we should have validating admission controllers to prevent such types to be created or updated. Let me know when you think differently.